data/content/agent_attributes.yaml

```diff
@@ -4,19 +4,25 @@ attributes:
   default_value: None
   required: true
   desc: |
-      The agent registration token from your organization’s _Agents_ page, used only to register new agents. To get your token, log in to Buildkite, navigate to _Agents_, then select _Reveal Agent Token_.
-- name: bootstrap-script
-  env_var: BUILDKITE_BOOTSTRAP_SCRIPT_PATH
-  default_value: "buildkite-agent bootstrap"
-  required: false
-  desc: |
-      Command to invoke the bootstrap process.
+      The agent registration token from your organization's _Agents_ page, used only to register new agents. To get your token, log in to Buildkite, navigate to _Agents_, then select _Reveal Agent Token_.
 - name: build-path
   env_var: BUILDKITE_BUILD_PATH
   default_value: "(depends on platform)"
   required: true
   desc: |
       Path to where the builds will run from.
+- name: allowed-repositories
+  env_var: BUILDKITE_ALLOWED_REPOSITORIES
+  default_value: "none"
```

Comment:
Question, is the `default_value` actually equal to `"none"`?

Reply:
nah, the default value will be `""`

Reply:
Thanks for that, sounds sensible!